### PR TITLE
Rewrite stream getting-started guide

### PR DESCRIPTION
**Pull request by sabbyanandan**
- Clean up the TODOs
- Revise the stream guide entirely
- Replace wrong screenshots 
- Try to make stream/batch look somewhat consistent

### CODE REVIEW

File: content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md [hunk 1]
```diff
@@ -157,59 +184,8 @@ Now you can delete the stream you created. To do so:
...
-To set specific platform properties, use the prefix `deployer.<application-name>.<platform-name>` where platform name is `local`, `cloudfoundry`, or `kubernetes`
-
-For example:
-
-```
-stream deploy http-ingest --properties "deployer.log.local.javaOpts=-Xmx2048m -Dtest=foo
-```
-
-**TODO: Need a reference to available properties, perhaps point to the reference guide**
+This information can be found in the [Stream Monitoring](%currentpath%/eature-guides/streams/monitoring/) guide.
```
Comment (cppwfs) [inline content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md]: bad link.  Should be feature guides.

File: content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md [hunk 2]
```diff
@@ -67,11 +66,7 @@ Now that you have defined a stream, you can deploy it. To do so:
 
 If you are using the local Data Flow Server, add the following deployment property to set the port to avoid a port collision.
 
-**TODO show in UI**
-
-```
-stream deploy http-ingest --properties app.http.server.port=9000
-```
+    ![Unique Port](images/dataflow-unique-port.png)
```
Comment (cppwfs) [inline content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md]: The indentation is showing this a code segment preventing image from showing up.

File: content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md [hunk 3]
```diff
@@ -157,59 +184,8 @@ Now you can delete the stream you created. To do so:
 
 ## Updating and Rolling back a Stream
 
-This information can be found in the [Continuous Delivery Basics Guide](%currentpath%/stream-developer-guides/continuous-delivery/cd-basics)
+This information can be found in the [Continuous Delivery](%currentpath%/stream-developer-guides/continuous-delivery) guide.
```
Comment (cppwfs) [inline content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md]: %currentpath% is not being replaced.   So the link is invalid.

File: content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md [hunk 4]
```diff
@@ -110,37 +104,70 @@ Once a stream is deployed, you can view its logs. To do so:
...
-    $ cf logs ticker-314-log-v1
-    ...
-    ...
-    2017-11-20T15:39:43.76-0800 [APP/PROC/WEB/0] OUT 2017-11-20 23:39:43.761  INFO 12 --- [ ticker-314.time.ticker-314-1] log-sink                                 : 11/20/17 23:39:43
-    2017-11-20T15:39:44.75-0800 [APP/PROC/WEB/0] OUT 2017-11-20 23:39:44.757  INFO 12 --- [ ticker-314.time.ticker-314-1] log-sink                                 : 11/20/17 23:39:44
-    2017-11-20T15:39:45.75-0800 [APP/PROC/WEB/0] OUT 2017-11-20 23:39:45.757  INFO 12 --- [ ticker-314.time.ticker-314-1] log-sink                                 : 11/20/17 23:39:45
+Now you can verify the logs.
 
+```bash
+cf logs ticker-314-log-v1
```
Comment (cppwfs) [inline content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md]: Should be http-ingest-314-log-v1

File: content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md (hunk 3 above)
Comment (sabbyanandan) [reply content/documentation/pages/3-stream-developer-guides/1-getting-started/1-stream.md]: I'm checking with @oodamien on this one

**Comment by cppwfs** [comment]
LGTM

**Comment by cppwfs** [comment]
Squashed Merged

**Pull request closed without merging**